src/blocks/row/edit.js

```diff
@@ -183,24 +187,26 @@ const Edit = ( props ) => {
 						: <>
 							<ButtonGroup aria-label={ __( 'Select row layout', 'coblocks' ) } className="block-editor-inner-blocks__template-picker-options block-editor-block-pattern-picker__patterns">
 								<Button
-									icon="exit"
 									className="components-coblocks-row-placeholder__back"
+									icon="exit"
+									label={ __( 'Back to columns', 'coblocks' ) }
 									onClick={ () => {
+										__unstableMarkNextChangeAsNotPersistent();
```

Comment:
I think these are only needed inside `useEffect`. Could you try without the ones that are not inside `useEffect` and see if it still works?